registry/polaris/registry.go

```diff
@@ -301,6 +301,10 @@ func createRegisterParam(url *common.URL, serviceName string) *api.InstanceRegis
 			Metadata:  metadata,
 		},
 	}
+
+	req.SetTTL(5)
```

Comment:
Why do you use 5 instead of other number here?

Reply:
Nacos default heartbeat reporting time at 5 seconds, so here is a direct reference

Reply:
Ok, could you define a constant and replace 5 with it here?

Reply:
ok